moduleframework/dockerlinter.py

```diff
@@ -236,10 +236,16 @@ def check_helpmd_is_present(self):
                  False if help.md is not specified in Dockerfile
         """
         helpmd_present = False
-        for c in self.docker_dict[COPY]:
-            if "help.md" in c:
-                helpmd_present = True
-        for a in self.docker_dict[ADD]:
-            if "help.md" in a:
-                helpmd_present = True
+        try:
```

Comment:
This can be implemented a bit simpler:

```python
try:
    for instruction in [COPY, ADD]:
        for a in self.docker_dict[instruction]:
            if "help.md" in a:
                return True
except KeyError:
    log.error("Instruction %s is not present in Dockerfile", instruction)
return False
```

Reply:
But this will fail if `help.md` does not exist in COPY tag always. Check for ADD is skipped at all, I will rewrite it a bit.

Reply:
oh, right, that's a good point, my bad